src/swap/sideshift_mock_data.js

```diff
@@ -0,0 +1,48 @@
+export const QUOTE_REQUEST_BODY = {
```

Comment:
Where is the mock data currently being used?

Reply:
not being used, removing.